mcenter_cli/parallelm/mcenter_cli/command_action_log.py

```diff
@@ -0,0 +1,16 @@
+from cliff.command import Command
+
+from parallelm.mcenter_cli.list_actions import list_actions
+
+
+class ActionLogListCommand(Command):
+    def get_parser(self, prog_name):
+        parser = super(ActionLogListCommand, self).get_parser(prog_name)
+        parser.add_argument('--object', action='store', help='List of Objects for which to get actions')
+        parser.add_argument('--action', action='store', help='List of actions to query')
+        parser.add_argument('--username', action='store', help='List of User, whose actions should be queried')
```

Comment:
"List of User" doesn't make sense to me. The other two do make sense -- List of objections and list of actions. Should this be "List of Users"?

Reply:
Fixed